### DISCUSSION TITLE
Client network socket disconnected before secure TLS connection was established

### DISCUSSION THREAD

**Discussion by narmadakankuri9** [Accenture/sfmc-devtools#1573] [Q&A]
I have installed the sfmc plugin and along with all prerequisites. I have executed the below commands to start a project. 

- mcdev init
- I gave git user name and email
- Client ID, client Secret and Base auth URL.
- When I click on enter it starts displaying **Client network socket disconnected before secure TLS connection was established**
- I'm able to access the sfmc URL.

<img width="956" alt="Screenshot 2024-08-15 112153" src="https://github.com/user-attachments/assets/6747a4da-2ac2-4d9e-bdf4-201f8ff9bac0">

**Comment by JoernBerkefeld** [comment]
Looks like an Internet problem to me. The server would more clearly state if this was caused by a bad call or insufficient credentials.
Have you tried this via mobile Hotspot to ensure it's not your local (company?) network blocking the request?
